src/pandapipes/create.py

```diff
@@ -501,20 +498,29 @@ def create_pipe_from_parameters(net, from_junction, to_junction, length_km, diam
     :rtype: int
 
     :Example:
-        >>> create_pipe_from_parameters(net, from_junction=0, to_junction=1,
-        >>>                             length_km=1, diameter_m=40e-3)
+        >>> create_pipe_from_parameters(net, from_junction=0, to_junction=1, length_km=1, diameter_m=40e-3)
 
     """
     add_new_component(net, Pipe)
 
     index = _get_index_with_check(net, "pipe", index)
     _check_branch(net, "Pipe", index, from_junction, to_junction)
 
+
+    #Check if alpha parameter was declared (Alpha has been changed to U)
+    if 'alpha_w_per_m2k' in kwargs:
+
+        if u_w_per_m2k == 0:
+            u_w_per_m2k = kwargs['alpha_w_per_m2k']
+
+        raise DeprecationWarning(' In the future alpha will be renamed to u')
```

Comment:
Raising a deprecation warning interrupts the code flow. What we need here is just an output. I would preferrably stick with the python standard of using the warnings module for this, instead of our normal logger. So what would be done here is:

    import warnings
    warnings.warn("The parameter alpha_w_per_m2k has been renamed to u_w_per_m2k."
                       "It will be removed in future.", DeprecationWarning)

Preferrably, we'd also like to make sure with some tests, that the behavior is as expected. This can be done with

    import pytest
    with pytest.warns(DeprecationWarning):
        pandapipes.create_pipe...(with alpha)
        assert ... # make sure that u is the given alpha value

Reply:
@dlohmeier I added two tests. What is your opinion?